lib/active_merchant/billing/gateways/card_stream.rb

```diff
@@ -334,7 +335,8 @@ def commit(action, parameters)
         # adds a signature to the post hash/array
         add_hmac(parameters)
 
-        response = parse(ssl_post(self.live_url, post_data(action, parameters)))
+        api_url = @custom_api_url || self.live_url
```

Comment:
Can you update this to be `@options[:custom_api_url]`? There's not need to set @custom_api_url it in initialize

Reply:
done!